Tests/UnitTests/Networking/Backend/BaseBackendTest.swift

```diff
@@ -41,8 +41,14 @@ class BaseBackendTests: TestCase {
         self.createDependencies(dangerousSettings: self.dangerousSettings)
     }
 
-    final func createDependencies(dangerousSettings: DangerousSettings? = nil,
-                                  storeKitVersion: StoreKitVersion = .default) {
+    final func createDependencies(dangerousSettings: DangerousSettings? = nil) {
+        // Need to force StoreKit 1 because we use iOS 13 snapshots
+        // for watchOS tests which contain StoreKit 1 headers
+        #if os(watchOS)
+        let storeKitVersion = StoreKitVersion.storeKit1
+        #else
+        let storeKitVersion = StoreKitVersion.default
```

Comment:
Since the default has changed, I guess we will be running these tests in SK2, which is probably ok, but wanted to make sure we don't need to cover SK1 behavior (other than in watchOS).

Reply:
Yeah that's what changed. Ideally we want separate snapshots for watchOS instead of reusing the ones from iOS 13. That's on my to-do list but I didn't want to add more files to this PR 😅